sdk/typescript/src/providers/json-rpc-provider.ts

```diff
@@ -327,7 +327,6 @@ export class JsonRpcProvider extends Provider {
   async getNormalizedMoveModulesByPackage(
     packageId: string,
   ): Promise<SuiMoveNormalizedModules> {
-    // TODO: Add caching since package object does not change
```

Comment:
Removed these since we're considering caching to be a dapp-level concern now. When we have a dapp SDK that provides APIs for these methods, we can set the cache time there.